src/lua/BabblerBomb/Bombler.lua

```diff
@@ -0,0 +1,284 @@
+		end
+	end
+	
+	oldUpdateBabbler(self, deltaTime)
+  
+end
+
+
+if Server then
+
+    local kEyeOffset = Vector(0, 0.2, 0)
+    function Bombler:GetEyePos()
+        return self:GetOrigin() + kEyeOffset
+    end
+
+
+
+    local function KillCallback(this)
+        this:Kill()
+        return -- remove callback
+    end
+
+    function Bombler:OnOwnerChanged(oldOwner, newOwner)
+		-- Destroy Babblers without Owner
+		-- Use callback to avoid server crashs due to calls to destroyed unit by mixins
+		--self:AddTimedCallback(KillCallback, 1)
+
+    end
+
+    function Bombler:OnEntityChange(oldId, newId)
+		if self.moveType == kBabblerMoveType.Cling then
+			self:SetMoveType(kBabblerMoveType.Move, nil, self:GetOrigin(), true)
+		end
+    end
+
+    function Bombler:GetBabblerBall()
+		return false
+    end
+
+
+ 	function Bombler:MoveRandom()
+		PROFILE("Babbler:MoveRandom")
+
+		if self.moveType == kBabblerMoveType.None and self:GetIsOnGround() and not self.babblerOffMap then
+			-- check for targets to attack
+			local target = self.targetSelector:AcquireTarget() or self:GetTarget()
+			local owner = self:GetOwner()
+			local alive = owner and HasMixin(owner, "Live") and owner:GetIsAlive()
+			local ownerOrigin = owner and (not owner:isa("Commander") and owner:GetOrigin() or owner.lastGroundOrigin)
+
+			if target then
+				-- All babblers get that attack order too (all the group focus on the same target)
+				for _, babbler in ipairs(GetEntitiesForTeamWithinRange("Bombler", self:GetTeamNumber(), self:GetOrigin(), 30)) do
+					if babbler:GetOwner() == owner and babbler:GetTarget() ~= target then
+						babbler:SetMoveType(kBabblerMoveType.Attack, target, target:GetOrigin())
+					end
+				end
+			
+			else
+				-- nothing to do, find something "interesting" (maybe glowing)
+				local targetPos = self:FindSomethingInteresting()
+
+				if targetPos then
+					self:SetMoveType(kBabblerMoveType.Move, nil, targetPos)
+				end
+			end
+
+			-- jump randomly
+			if math.random() < 0.6 then
+				self:JumpRandom()
+			end
+		end
+
+		return not self.clinged and self:GetIsAlive()
+	end
+
+
+
+    local function GetCanReachTarget(self, target)
+
+        local obstacleNormal = Vector(0, 1, 0)
+
+        local targetOrig = target:GetOrigin()
+        local targetTraceOrigin = HasMixin(target, "Target") and target:GetEngagementPoint() or target:GetOrigin()
+        local trace = Shared.TraceRay(self:GetOrigin() + kEyeOffset, targetTraceOrigin, CollisionRep.LOS, PhysicsMask.All, EntityFilterAll())
+
+        local canReach = trace.fraction >= 0.9 or self:IsTargetReached(targetOrig, 1)
+        if canReach then
+            return true, nil
+        else
+            obstacleNormal = trace.normal
+        end
+
+        return false, obstacleNormal
+
+    end
+
+    function Bombler:UpdateCling(deltaTime)
+      --[[  if not self:Attach(deltaTime) then
+            self:Detach()
+            return false
+        end
+
+        return true]]--
+    end

```

Comment:
I would suggest overriding `Babble.SetMoveType` so the kBabblerMoveType.Cling cannot be set for the Bombler class. That in combination with overriding `MoveUpdate` should ensure they never attach to anything.